src/IIIFPresentation/API/Program.cs

```diff
@@ -88,7 +88,11 @@
     app.UseSwaggerUI();
 }
 
+var rewriteOptions = new RewriteOptions()
+    .AddRedirect("(.*)/$", "$1");
```

Comment:
would it better to use `.AddRewrite` over `.AddRedirect`? Would save the client from having to make another call

Reply:
One is telling customer to call properly, the other opens a pandoras box :D